README.md

```diff
@@ -23,6 +23,87 @@ gem install optimizely-sdk
+
+
+~~~~~~
+ http_project_config_manager = Optimizely::HTTPProjectConfigManager.new(
+        sdk_key: nil,
+        url: nil,
+        datafile: nil,
+        url_template: nil,
+        auto_update: nil,
+        polling_interval: nil,
+        start_by_default: nil,
+        blocking_timeout: nil,
+        logger: nil,
+        error_handler: nil,
+        skip_json_validation: false,
+        notification_center: notification_center
+      )
+~~~~~~   
+**Note**: One of the sdk_key or url must be provided. When both are provided, url takes the preference.
+
+**sdk_key**
+The sdk_key is used to compose the outbound HTTP request to the default datafile location on the Optimizely CDN.
+
+**datafile**
+You can provide an initial datafile to bootstrap the  `DataFileProjectCongig`  so that it can be used immediately. The initial datafile also serves as a fallback datafile if HTTP connection cannot be established. The initial datafile will be discarded after the first successful datafile poll.
+
+**polling_interval**
+The polling_interval is used to specify a fixed delay in seconds between consecutive HTTP requests for the datafile.
+
+**url_template**
+A string with placeholder `{sdk_key}` can be provided so that this template along with the provided sdk key is used to form the target URL.
+
+**start_by_default**
+Boolean flag used to start the AsyncScheduler for datafile polling if set to True.
+
+**blocking_timeout**
+Maximum time in seconds to block the get_config call until config has been initialized.
+
+You may also provide your own logger, error handler or notification center. 
+
+
+#### Advanced configuration
+The following properties can be set to override the default configurations for HTTPConfigManager.
+
+| **PropertyName** | **Default Value** | **Description**
+| -- | -- | --
+| update_interval | 5 minutes | Fixed delay between fetches for the datafile 
+| sdk_key | nil | Optimizely project SDK key
+| url | nil | URL override location used to specify custom HTTP source for the Optimizely datafile. 
+| url_template | 'https://cdn.optimizely.com/datafiles/{sdk_key}.json' | Parameterized datafile URL by SDK key.
+| datafile | nil | Initial datafile, typically sourced from a local cached source.
+| auto_update | true | Boolean flag to specify if callback needs to execute infinitely or once only.
+| start_by_default | true | Boolean flag to specify if datafile polling should start right away as soon as the HTTPConfigManager initializes
+| blocking_timeout | 15 seconds | Maximum time in seconds to block the get_config call until config has been initialized.
+
+A notification signal will be triggered whenever a _new_ datafile is fetched and Project Config is updated. To subscribe to these notifications you can use the `notification_center.add_notification_listener(Optimizely::NotificationCenter::NOTIFICATION_TYPES[:OPTIMIZELY_CONFIG_UPDATE], @callback)`
```

Comment:
Suggest deleting "you can" and adding a comma:
...these notifications, use the...